hazelcast/src/main/java/com/hazelcast/map/impl/NearCache.java

```diff
@@ -93,21 +96,21 @@ public int compare(CacheRecord o1, CacheRecord o2) {
         }
     };
 
-    private SizeEstimator nearCacheSizeEstimator;
+    private final SizeEstimator nearCacheSizeEstimator;
```

Comment:
So far they have only single instance of size estimator so it's ok to keep it as is.

I don't want to do object traverse whenever not nessecary